src/redux/services/helm.ts

```diff
@@ -208,3 +278,55 @@ export function findContainingHelmCharts(helmChartMap: HelmChartMapType, fileEnt
     });
   return charts;
 }
+
+export const getObjectKeys = (obj: any, prefix = ''): string[] =>
```

Comment:
We have a library called `flat` installed. I think it does the same as you are trying to do here yourself.

See https://www.npmjs.com/package/flat.

Reply:
Thanks for the library, it would work usually but if I have an object like this:

`{
    key1: {
        keyA: 'valueI'
    },
    key2: {
        keyB: 'valueII'
    },
    key3: { a: { b: { c: 2 } } }
}`
I expect it to return 
`['key1', 'key1.keyA']`

Basically I want it to return all the elements not just the bottom leafs. I need this because in some cases we have object or array references in helm templates

Reply:
Would you mind adding a small JavaDoc with this example? That way its intend is clear and what makes it special.